dpr1fact.c

```diff
@@ -365,7 +365,7 @@ char dodpr1fact(double *beta, mwIndex *perm, double *d, double t, const double *
    ------------------------------------------------------------ */
   else{
     psqrdep = 0.0;
-    for(i = 0; dep[i] < m; i++)
+    for(i = 0, j = 0; dep[i] < m; i++)
```

Comment:
I don't like this change.  Please set `j` in a separate line before for clarity.